### PR TITLE
:sparkles: Add default release number

### PR DESCRIPTION
**Pull request by flaxel**
### Type of Change

- [ ] Bugfix
- [x] Enhancement / new feature
- [ ] Refactoring
- [ ] Documentation

### Description

- add default release number for drafter with format `YEAR.WEEK.PATCH`

### Checklist

- [x] Add relevant labels (for example type of change or patch/minor/major)
- [x] Make sure not to introduce some mistakes
- [x] Update documentation
- [x] Review the [Contributing Guideline](../blob/main/CONTRIBUTING.md) and sign CLA
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

File: .github/workflows/template_release_drafter.yml [hunk 1]
```diff
@@ -20,16 +20,39 @@ on:
 jobs:
   update_release_draft:
 
-    name: update release draft
+    name: Update Release Drafter
```
Comment (timdittler) [inline .github/workflows/template_release_drafter.yml:23]:
```suggestion
    name: Update Release Draft
```
We're updating the draft, not the drafter :D

**Review by timdittler — APPROVED** [review]

File: .github/workflows/template_release_drafter.yml [hunk 2]
```diff
@@ -20,16 +20,39 @@ on:
...
     steps:
+      - name: checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: set version to github env
+        id: set_version
+        run: |
+          # fetch last tag from git
+          OLD_VERSION=`git describe --tags $(git rev-list --tags --max-count=1)`
```
Comment (soemo) [inline .github/workflows/template_release_drafter.yml:35]: In the studio repository we use the command `git tag | sort -V | tail -1` https://github.com/Staffbase/experience-studio/blob/99001ccabb17aac0f1c36fc0779c471d9f889542/.github/scripts/version.js#L4

**Review by 0x46616c6b — COMMENTED** [review]
I'm not sure if we should implement the determination of the next version in a fixed format for all repositories using that workflow. That mean they need to checkout the repository another time with full fetch depth. This use-case is specific to a sub-set of our internal services and this workflow should be as agnostic as possible. 

We should rethink if we can implement an action/workflow that provides that version information and we can put two workflows together. Would be more useful and agnostic in my eyes.

**Comment by flaxel** [comment]
@0x46616c6b [With the new version of the checkout action](https://github.com/actions/checkout/pull/579), this is no longer necessary, so the use case is a bit more cost-effective. I'm just waiting for a new release. 🚀 

Otherwise, we could check in the step if one of the parameters is set and if it should be the case, then we can also omit the checkout and everything else. 💡 

But I don't know if it's valuable to have a completely new reusable workflow. WDYT? 🤔

**Comment by flaxel** [comment]
**TODO**: 

- create a new reusable workflow for versioning number
- check if release drafter configuration could be omitted

File: .github/workflows/template_release_drafter.yml (hunk 2 above)
Comment (flaxel) [reply .github/workflows/template_release_drafter.yml:35]: The disadvantage of this command is that you sort it alphabetically. 🤔 So if you change the version structure you could run into issues. With the command above you won't have this issue. 🙌

**Comment by flaxel** [comment]
closed in favor of #155

**Pull request closed without merging**